standalone/app/controllers/OAuthLoginController.scala

```diff
@@ -100,3 +106,26 @@ object OAuthLoginController extends Controller with ExecutionContexts with impli
     Redirect(routes.OAuthLoginController.login()).withNewSession
   }
 }
+
+object AuthCookie extends Logging {
+
+  private val cookieName = "GU_PV_AUTH"
+  private val thirtyDaysInSeconds: Int = 2592000
```

Comment:
As discussed even 30days scares me from a security perspective. But seeing as its only preview i'm not that bothered...